README.md

```diff
@@ -26,3 +26,45 @@ Delete Loadflow results definitively :
 ```py
 python3 delete_computation_results.py -lf
 ```
+
+## Grafana scripts
+### Import existing alert-rules JSON
+First, we need to create a folder.
+Execute this command :
+```py
+python3 grafana_create_folder.py {{FOLDER_NAME}}
+```
+
+This command should return a json with a field "uid" which will be {{FOLDER_UID}} for the next command
+
+Then we can create alert-rules in the previously created folder.
+Execute this command :
+```py
+python3 grafana_create_alerts.py -f {{PATH_TO_ALERT_RULES_1}} -f {{PATH_TO_ALERT_RULES_2}} -d {{DATASOURCE_UID}} -p {{FOLDER_UID}}
```

Comment:
We can document "-i" the rule group interval?

Reply:
fixed